src/Client/BatchRequestFailureException.php

```diff
@@ -16,20 +15,17 @@ class BatchRequestFailureException extends Exception
      */
     private $batchRequest;
     /**
-     * @var BatchResults
+     * @var array
      */
     private $batchResults;
 
     /**
      * @param BatchRequest $batchRequest
-     * @param BatchResults $batchResults
+     * @param array $batchResults
      */
-    public function __construct(BatchRequest $batchRequest, BatchResults $batchResults)
+    public function __construct(BatchRequest $batchRequest, array $batchResults)
```

Comment:
it's array of Response or Exception object. Yeah, probably I should implement BatchResults object if it's not in Guzzle any more.